monai/auto3dseg/analyzer.py

```diff
@@ -245,9 +245,11 @@ def __call__(self, data):
         report[ImageStatsKeys.SHAPE] = [list(nda.shape) for nda in ndas]
         report[ImageStatsKeys.CHANNELS] = len(ndas)
         report[ImageStatsKeys.CROPPED_SHAPE] = [list(nda_c.shape) for nda_c in nda_croppeds]
-        report[ImageStatsKeys.SPACING] = np.tile(
-            np.diag(data[self.image_meta_key]["affine"])[:3], [len(ndas), 1]
-        ).tolist()
+        report[ImageStatsKeys.SPACING] = (
+            affine_to_spacing(data[self.image_key].affine).tolist()
```

Comment:
this is a useful fix... would be great if it could be in v1.0.1 (releasing next Monday)